assets/js/app/Data.js

```diff
@@ -183,10 +183,10 @@ LocusZoom.Data.Source.prototype.getData = function(state, fields, outnames, tran
 };
 
 
-LocusZoom.Data.Source.prototype.parseResponse  = function(x, chain, fields, outnames, trans) {
-    var records = this.parseData(x.data || x, fields, outnames, trans);
-    var res = {header: chain.header || {}, body: records};
-    return res;
+LocusZoom.Data.Source.prototype.parseResponse = function(resp, chain, fields, outnames, trans) {
```

Comment:
@MrFlick - okay, take a look at this now. I refactored it once more to be a bit more in keeping with the original approach. The type checking is important because the StaticJSON data source uses the default parse method, and it's passing through an object, not a string representing an object (and `JSON.parse()` fails when fed an object as it casts its argument to a string, resulting in "[object Object]").